### PR TITLE
Add support for toggling raise hand and pin presentation

### PR DESCRIPTION
**Pull request by gototen**
Introduce the ability to toggle (mute/unmute) any arbitrary element.

~Unfortunately, I could not force Google Meet to load in German, so I could not find the matching `de` strings.~

### CODE REVIEW

File: browser-extension/contentScript.js [hunk 1]
```diff
@@ -47,6 +47,16 @@ const SidepanelParticipantsButtonLabels = [
...
 ]
+const HandButtonLabels = [
+  // This is the raise hand button
+  ["Raise hand", "Lower hand"],        // English
+  ["Raise hand", "Lower hand"],        // TODO: Replace with the equivalent strings in German
+];
+const PinButtonLabels = [
+  // This is the pin presentation button
+  ["presentation to your main screen", "presentation to your main screen"], // English
+  ["presentation to your main screen", "presentation to your main screen"], // TODO: Replace with the equivalent strings in German
```
Comment (gototen) [inline browser-extension/contentScript.js:58]: @woehrl01 Would you be able to find the equivalent german strings for `Raise hand`, `Lower hand` and `Pin <user>'s presentation to your main screen` in Google Meet?  Thank you!

Comment (woehrl01) [reply browser-extension/contentScript.js:58]: @gototen The strings are "Melden", "Meldung zurückziehen" and "<user> an meinen Hauptbildschirm anpinnen." and "Loslösen" (for unpinning).

Comment (gototen) [reply browser-extension/contentScript.js]: Thank you!

**Comment by ChrisRegado** [comment]
Thanks for posting this!

I merged it in, integrated it with a (fairly large) browser extension refactor I was working on to support different languages (https://github.com/ChrisRegado/streamdeck-googlemeet/issues/7), and published a new release.

One thing to note is I was having issues with the Disconnected state on the pin button, so I removed/hid that for now and re-used the "pin off" image, just like the Hand button does. It was proving difficult to get it to accurately reflect whether or not there was an active presentation under cases such as joining a meeting with an already-active presentation, or when you're on meet.google.com but not in a meeting yet, etc. If anyone finds a good fix for that, please let me know (or post a PR)!

**Pull request merged**